stairsplus/common.lua

```diff
@@ -49,7 +49,7 @@ stairsplus.register_single = function(category, alternate, info, modname, subnam
 			def.description = desc_base .. alternate:gsub("_", " "):gsub("(%a)(%S*)", function(a, b) return a:upper() .. b end)
 		end
 	end
-	def.groups = stairsplus:prepare_groups(fields.groups)
+	def.groups = stairsplus:prepare_groups(fields.groups, disabled_in_creative_inventory)
```

Comment:
Could be replaced by a generic setting check: (requires settings to use the singular form)
```
def.groups = stairsplus:prepare_groups(fields.groups)
if moreblocks.config["stairsplus_" .. category .. "_not_in_creative_inventory"]  then
	def.groups.not_in_creative_inventory = 1
end
```